src/javascript/app/japan/cashier.js

```diff
@@ -17,11 +17,11 @@ const CashierJP = (() => {
             } else {
                 BinarySocket.send({ get_account_status: 1 }).then((response_status) => {
                     if (!response_status.error && /cashier_locked/.test(response_status.get_account_status.status)) {
-                        $container.find('#cashier_locked_message').html('Your cashier is locked.').setVisibility(1); // Locked from BO
+                        $container.find('#cashier_locked_message').text(localize('{JAPAN ONLY}Your cashier is locked.')).setVisibility(1); // Locked from BO
```

Comment:
do we need to add these to the generate static data file?

Reply:
`Your cashier is locked.` is already in generate static data tho, should we remove `{JAPAN ONLY}` tag here instead

Reply:
sure

Reply:
ok :)